docker-compose.yml

```diff
@@ -128,12 +128,25 @@ services:
     ports:
       - '8078:8078'
 
+  # viz:
+  #   image: viz
+  #   container_name: viz
+  #   volumes:
+  #     - $TP_VIZ_DIR:/app:cached
+  #   environment:
+  #     NODE_ENV: development
+  #   expose:
```

Comment:
`expose` isn't required if viz isn't needed by any other service. The user can already get to the port via the `ports` directive.